src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -55,11 +53,17 @@
     @Column(name = "inship_registration_end_date")
     private LocalDate registrationEndDate;
 
-//    private List<FormFullDTO> formList;
-//    private List<UserDTO> techList;
-//    private List<UserDTO> adminList;
-//    private List<LocationDTO> countryList;
-//    private InternshipFormat InternshipFormat;
-//    private List<Technology> technologyList;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "inship_format_name", nullable = false)
+    private InternshipFormat InternshipFormat;
+
+    @ManyToOne
+    @JoinColumn(name="inship_technology_name", nullable=false)
+    private Technology technology;
```

Comment:
Just note: Did you investigate posibility to use  @ElementCollection

Reply:
done 
![image](https://user-images.githubusercontent.com/65280608/114389871-3918d980-9b9e-11eb-9c81-253225a7e1e1.png)

Reply:
![image](https://user-images.githubusercontent.com/65280608/114399517-ac741880-9ba9-11eb-8861-70acff354406.png)